cypress/features/regression/flatTable.feature

```diff
@@ -0,0 +1,89 @@
+Feature: FlatTable component
+  I want to check Flat Table component properties
```

Comment:
We should unify how we use component name `Flat Table` or this way `FlatTable`.

Reply:
fixed